docs/3.0.0-beta.x/guides/schedule-article.md

```diff
@@ -0,0 +1,73 @@
+# Schedule article
+
+This guide will explain how to create an article schedule system.
+
+## Introduction
+
+This guide will use this [draft system](.draft.md) guide. So first read it to understand the draft/published status and the Article API that we will use in this guide.
```

Comment:
This guid will continue from the draf system guide. Make sure to read it to understand ...


should be `./draft.md`